### ISSUE TITLE
Library not supporting Android 12

### ISSUE THREAD

**Issue by mariobat88** [plaid/plaid-link-android#191]
# The problem

Library not supporting Android API 12. Looks like the com.plaid.internal.LinkRedirectActivity doesn't have the exported = "true" defined although it has intent filters.

# Environment

|                        |  |
| ------------------------| ---------- |
| Android OS Version                    | e.g. 12.0 (31)                |
| Android Devices/Emulators    | e.g. Any               |

# Steps to Reproduce
Just include the Plaid library to any project and try to build the app via compileSdk 31.

# Expected Result
App would compile successfuly.

# Screenshots
This is what I see in my APK file when I open the Manifest there, so you can see "exported" is not added:
```
<activity
            android:theme="@ref/0x0103022e"
            android:name="com.plaid.internal.LinkRedirectActivity">

            <intent-filter>

                <action
                    android:name="android.intent.action.VIEW" />

                <category
                    android:name="android.intent.category.DEFAULT" />

                <category
                    android:name="android.intent.category.BROWSABLE" />

                <data
                    android:scheme="plaid"
                    android:host="redirect" />
            </intent-filter>
        </activity>
```

**Comment by amytang0** [comment]
Thanks for flagging this! This should be fixed in our next release 3.6.0 in a few days, but in the meantime you can override this in your `AndroidManifest.xml`, e.g.

```
<!--    overriding link SDK activities to support Android 12 by adding android:exported -->
        <activity
            android:name="com.plaid.internal.LinkRedirectActivity"
            android:theme="@android:style/Theme.Material.NoActionBar"
            android:exported="true">
            <intent-filter>
                <action android:name="android.intent.action.VIEW" />

                <category android:name="android.intent.category.DEFAULT" />
                <category android:name="android.intent.category.BROWSABLE" />

                <data
                    android:host="redirect"
                    android:scheme="plaid" />
            </intent-filter>
        </activity>
        ```

**Comment by amytang0** [comment]
Closing this due to stale activity, 3.6.0 is released and supports Android 12
